src/main/java/service/FillStorage.java

```diff
@@ -0,0 +1,8 @@
+package service;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface FillStorage {

```

Comment:
Be attentive with [class](https://mate-academy.github.io/style-guides/java/java.html#s5.2.2-class-names) and [method](https://mate-academy.github.io/style-guides/java/java.html#s5.2.3-method-names) naming.
```suggestion
public interface FillStorage {
```
Let's fix all services and methods names. The class name must be a noun, method name - a verb